src/App.js

```diff
@@ -53,9 +53,11 @@ class App extends Component {
         const tags = {};
 
         for(let project of data) {
-          categories[project.category] = "";
+          let category = project.category.split(',')
```

Comment:
here `category` could potentially be something like `["Job Training and Opportunity", " Childhood Support", " Education"]`

When working with the the result of `split(',')` could you treat it like its an array with potentially more than 1 element? Most of the time its just 1 thing but it isn't always.